### DISCUSSION TITLE
Memory usage test

### DISCUSSION THREAD

**Discussion by GabrieleCicconetti** [socketio/socket.io#3666] [General] [answered]
Hi :). How can I test my app's memory usage on the server? Is there any tool to simulate a large number of simultaneous connections? I would like to do it to be able to choose the best hosting for my needs. Thank you.

**Comment by szarkowicz** [comment, marked as answer]
@GabrieleCicconetti  - [Artillery](https://artillery.io/blog/load-testing-socketio-with-artillery) - I have used this in the past to load test socket.io

**Reply by GabrieleCicconetti** [reply]
Wow thank you !
